pyproject.toml

```diff
@@ -17,7 +17,7 @@ requires-python = ">=3.11,<3.12"
 dynamic = ["version"]
 license = {file = "LICENSE.txt"}
 dependencies = [
-    "pandas>=1.5,<2.0",
+    "pandas>=1.5,<2.0.1",
```

Comment:
Allowing pandas 2.0+ will have no effect in this repo until `pudl` also allows 2.0+ -- so it's not really being tested yet. Which means there could be things hiding in this repo that are incompatible with pandas 2.0 that won't crop up until we switch `pudl` over to using pandas 2.0.